dbt/include/databricks/macros/relations/components/comment.sql

```diff
@@ -3,3 +3,19 @@
   COMMENT '{{ comment }}'
 {%- endif -%}
 {%- endmacro -%}
+
+
+{%- macro get_create_column_comment(model) -%}
+  {%- if model.columns -%}
+    {%- set column_definitions = [] -%}
+    {%- for column_name, column in model.columns.items() -%}
+      {%- set data_type = column.data_type or 'STRING' -%}
```

Comment:
Does this actually work?  In general we don't know the column data type, and I'm concerned that specifying as string will break when the columns are string type.

Reply:
Yes you're right, it breaks! Unfortunately, for this to work it we need to explicitly define data_types in the schema file of the respective model.

Reply:
@benc-db I have added a workaround where we initially create a temp view to infer the schema and utilize that to get the data type if undefined. Please let me know your thoughts! 😄  I have tested it in my local setup (1. with schema file and defined data_types 2. with schema file without defining any data types 3. without schema file) and works for both ST/MV

Reply:
This is an interesting approach.  I want to talk to dbt Labs about this to get their perspective, and I definitely will want to put behind a behavior flag (new tech recently introduced into dbt) because this will add some latency, but might be worth the trade off.